bin/phase-functions.sh

```diff
@@ -666,7 +666,8 @@ __dyn_install() {
 		ASFLAGS CBUILD CC CFLAGS CHOST CTARGET CXX \
 		CXXFLAGS EXTRA_ECONF EXTRA_EINSTALL EXTRA_MAKE \
 		LDFLAGS LIBCFLAGS LIBCXXFLAGS QA_CONFIGURE_OPTIONS \
-		QA_DESKTOP_FILE QA_PREBUILT PROVIDES_EXCLUDE REQUIRES_EXCLUDE ; do
+		QA_DESKTOP_FILE QA_PREBUILT PROVIDES_EXCLUDE REQUIRES_EXCLUDE \
+		PKG_INSTALL_MASK ; do
```

Comment:
Looks like we're not reading back this PKG_INSTALL_MASK value at uninstall time yet?

Reply:
Nevermind, PKG_INSTALL_MASK filters the file set, so we don't need to account for it at uninstall time.